.github/workflows/ci.yaml

```
@@ -17,6 +18,127 @@ jobs:
+      run: |
+        sshpass ssh -o StrictHostKeyChecking=no -i  mlrun.pem ${{ secrets.USERNAME }}@${{ secrets.IP }} <<EOF
+        # CLEANUP
+        echo "helm uninstall mlrun-ce"
+        helm uninstall -n mlrun mlrun-ce || echo true
+        echo "helm uninstall monitoring"
+        helm uninstall -n mlrun monitoring || echo true
+        echo "delete pv, pvc from mlrun"
+        kubectl delete all --all -n mlrun
+        echo "deleting mlrun namepsace";
+        kubectl delete namespace mlrun || echo true
+        EOF
+        
+    - name: Creating mlrun namespace
+      run: |
+        sshpass ssh -o StrictHostKeyChecking=no -i  mlrun.pem ${{ secrets.USERNAME }}@${{ secrets.IP }} <<EOF
+        echo "create mlrun namepsace"
+        kubectl create namespace mlrun        
+        echo "k8s namespaces - $(kubectl get namespace)"
+        EOF
+        
+    - name: Creating docker registry secret
+      run: |
+        sshpass ssh -o StrictHostKeyChecking=no -i  mlrun.pem ${{ secrets.USERNAME }}@${{ secrets.IP }} <<EOF
+        kubectl --namespace mlrun create secret docker-registry registry-credentials \
+        --docker-server ${{ secrets.DOCKER_SERVER }} \
+        --docker-username ${{ secrets.DOCKER_USERNAME }} \
+        --docker-email=${{ secrets.DOCKER_EMAIL }} \
+        --docker-password=${{ secrets.DOCKER_PASSWORD }}
+        EOF
+
+    - name: CE installation
+      run: |    
+        # CE INSTALLATION
+        sshpass ssh -o StrictHostKeyChecking=no -i  mlrun.pem ${{ secrets.USERNAME }}@${{ secrets.IP }} <<EOF
+        echo $(pwd)
+        cd ce/ce/charts/mlrun-ce
+        echo $(ls)
+        echo "helm dependency update"
+        helm dependency update
+        echo "installing community edition"
+        helm install mlrun-ce --namespace mlrun \
+        --set minio.resources.requests.memory="128Mi" \
+        --set  jupyterNotebook.persistence.size="1Gi" \
+        --set  mlrun.db.persistence.size="1Gi" \
+        --set  mlrun.api.persistence.size="1Gi" \
+        --set  mpi-operator.deployment.create="true" \
+        --set  mlrun.httpDB.dbType="sqlite" \
+        --set  mlrun.httpDB.dirPath="/mlrun/db" \
+        --set  mlrun.httpDB.dsn="sqlite:////mlrun/db/mlrun.db?check_same_thread=false"  \
+        --set  mlrun.httpDB.oldDsn=""  \
+        --set global.registry.url=${{ secrets.DOCKER_URL }}  --set global.registry.secretName=registry-credentials   \
+        --set global.externalHostAddress=${{ secrets.IP }}  \
+        --set nuclio.dashboard.externalIPAddresses[0]=${{ secrets.IP }}  \
+        --wait --debug --timeout=1h .
+        EOF
+
+    - name: Run demos from Jupyter pod notebook - news_article_nlp
+      uses: appleboy/ssh-action@v0.1.10
+      with:
+        host: ${{ secrets.IP }}
+        username: ${{ secrets.USERNAME }}
+        key: ${{ secrets.MLRUN_PEM }}
+        script: |
+          jupyter_service=$(kubectl -n mlrun get pods -o name | grep jupyter | head -n 1 | cut -d / -f 2)
```

```suggestion
          jupyter_pod=$(kubectl -n mlrun get pods -o name | grep jupyter | head -n 1 | cut -d / -f 2)
```
service is confusing as Kubernetes service means something else.
Same to all others